lib/lm_docstache/hide_custom_tags.rb

```diff
@@ -64,7 +64,8 @@ def replace_style(run_node)
         w_color.unlink if w_color
         style << "<w:color w:val=\"#{font_color}\"/>"
       else
-        run_node << "<w:rPr><w:color w:val=\"#{font_color}\"/></w:rPr>"
+        text_node = run_node.at_css('w|t')
+        text_node.add_previous_sibling("<w:rPr><w:color w:val=\"#{font_color}\"/></w:rPr>") if text_node
```

Comment:
Why not using the [Nokogiri::XML::Node#prepend_child](https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri/XML/Node#prepend_child-instance_method) method? It's simpler:
```suggestion
        run_node.prepend_child("<w:rPr><w:color w:val=\"#{font_color}\"/></w:rPr>")
```